### PR TITLE
DOC-846 Google OAuth2 Documentation on Consent Screen: improved documenation with reference to user types in consenst screen

### PR DESCRIPTION
**Pull request by mariaremote**
Hi Debs!

Here are my documentation suggestions for the linear ticket [DOC-846](https://linear.app/n8n/issue/DOC-846/google-oauth2-documentation-on-consent-screen).
I tried to make the two OAuth guides (**Single Service** vs. **Generic**) a bit more consistent in terms of headings etc. 
And I made the section on setting the `User Type` in the OAuth Consent Screen a bit more explanatory. 

I look forward to your feedback! 😊 
~ Ria

### CODE REVIEW

File: docs/integrations/builtin/credentials/google/oauth-generic.md [hunk 1]
```diff
@@ -49,6 +49,8 @@ This document contains instructions for creating a generic OAuth2 Google credent
 		
 5. Select **CREATE**.
 6. Enable each Google service API that you want to use:
+
+	1. If using Google Perspective or Google Ads: [Request API Access for Perspective](https://developers.perspectiveapi.com/s/docs-get-started){:targe=_blank .external-link} or a [Developer Token for Ads](https://developers.google.com/google-ads/api/docs/first-call/dev-token){:targe=_blank .external-link}.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/google/oauth-generic.md:53]:
```suggestion
	1. If using Google Perspective or Google Ads: [Request API Access for Perspective](https://developers.perspectiveapi.com/s/docs-get-started){:target=_blank .external-link} or a [Developer Token for Ads](https://developers.google.com/google-ads/api/docs/first-call/dev-token){:target=_blank .external-link}.  
```

Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/google/oauth-generic.md:53]:
```suggestion
	1. If using Google Perspective or Google Ads: [Request API Access for Perspective](https://developers.perspectiveapi.com/s/docs-get-started){:target=_blank .external-link} or a [Developer Token for Ads](https://developers.google.com/google-ads/api/docs/first-call/dev-token){:target=_blank .external-link}.  
```

File: docs/integrations/builtin/credentials/google/oauth-generic.md [hunk 2]
```diff
@@ -100,3 +104,6 @@ n8n doesn't support all scopes. When creating a generic Google OAuth2 API creden
 
 --8<-- "_snippets/integrations/builtin/credentials/google/unverified-app.md"
 
+### Google Cloud app becoming unauthorized
+
+For Google Cloud apps with **Publishing status** set to **Testing** and **User type** set to **External**, consent and tokens expire after seven days. Refer to [Google Cloud Platform Console Help | Setting up your OAuth consent screen](https://support.google.com/cloud/answer/10311615?hl=en#zippy=%2Ctesting){:target=_blank .external-link} for more information. To resolve this, reconnect the app in the n8n credentials modal.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/google/oauth-generic.md:109]:
```suggestion
For Google Cloud apps with **Publishing status** set to **Testing**, and **User type** set to **External**, consent and tokens expire after seven days. Refer to [Google Cloud Platform Console Help | Setting up your OAuth consent screen](https://support.google.com/cloud/answer/10311615?hl=en#zippy=%2Ctesting){:target=_blank .external-link} for more information. To resolve this, reconnect the app in the n8n credentials modal.
```

File: docs/integrations/builtin/credentials/google/oauth-single-service.md [hunk 3]
```diff
@@ -30,7 +39,7 @@ In Google Cloud:
 
 	??? Details "View detailed steps"
 		1. Select **OAuth consent screen**.
-		2. For **User Type**, select **Internal**.
+		2. As **User Type**, select **Internal** for user access within your organisation's Google workspace or **External** for any user with a Google account.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/google/oauth-single-service.md:42]:
```suggestion
		2. For **User Type**, select **Internal** for user access within your organization's Google workspace or **External** for any user with a Google account.
```

File: docs/integrations/builtin/credentials/google/oauth-single-service.md [hunk 4]
```diff
@@ -60,6 +69,8 @@ In Google Cloud:
 	1. If using Google Perspective or Google Ads: [Request API Access for Perspective](https://developers.perspectiveapi.com/s/docs-get-started){:targe=_blank .external-link} or a [Developer Token for Ads](https://developers.google.com/google-ads/api/docs/first-call/dev-token){:targe=_blank .external-link}.
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/google/oauth-single-service.md:69]:
```suggestion
	1. If using Google Perspective or Google Ads: [Request API Access for Perspective](https://developers.perspectiveapi.com/s/docs-get-started){:target=_blank .external-link} or a [Developer Token for Ads](https://developers.google.com/google-ads/api/docs/first-call/dev-token){:target=_blank .external-link}.  
```

File: docs/integrations/builtin/credentials/google/oauth-generic.md [hunk 5]
```diff
@@ -100,3 +104,6 @@ n8n doesn't support all scopes. When creating a generic Google OAuth2 API creden
 
 --8<-- "_snippets/integrations/builtin/credentials/google/unverified-app.md"
 
+### Google Cloud app becoming unauthorized
```
Comment (StarfallProjects) [inline docs/integrations/builtin/credentials/google/oauth-generic.md:107]: For this section on both pages, it looks like the content is identical? So let's turn it into a snippet. Snippets let us write once and reuse everywhere, which makes it easier to update in future.

To add a snippet:

1. Create a file in `_snippets`, at the correct location (the `_snippets` structure largely mirrors the site structure, so in this case, create it in the google creds snippets directory)
2. Add the snippet where you want to use it, like this: `--8<-- "_snippets/integrations/builtin/credentials/google/enable-apis.md"`

In this case, please create a snippet for "Google Cloud app becoming unauthorized", and add it on both pages, in the same way as I did for "Google hasn't verified this app"

The reason we don't include the section heading in the snippet: it makes it easier for writers, as we can still see the structure of the page, and know at a glance roughly what the snippet is about.

**Review by StarfallProjects — CHANGES_REQUESTED** [review]
Thanks for this! Please see the comments for details of changes. Let me know if you have any questions!

File: docs/integrations/builtin/credentials/google/oauth-single-service.md (hunk 4 above)
Comment (mariaremote) [reply docs/integrations/builtin/credentials/google/oauth-single-service.md:69]: This part was copied from https://docs.n8n.io/integrations/builtin/credentials/google/oauth-generic/
So need to adjust the `target` there too! (will make another PR for this)

**Review by StarfallProjects — APPROVED** [review]

**Pull request merged**